compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/ComposeWindow.uikit.kt

```diff
@@ -88,10 +88,39 @@ internal actual class ComposeWindow : UIViewController {
     private val keyboardOverlapHeightState = mutableStateOf(0f)
     private val safeAreaState = mutableStateOf(IOSInsets())
     private val layoutMarginsState = mutableStateOf(IOSInsets())
+
+    /*
+     * Initial value is arbitarily chosen to avoid propagating invalid value logic
+     * It's never the case in real usage scenario to reflect that in type system
+     */
     private val interfaceOrientationState = mutableStateOf(
-        InterfaceOrientation.getStatusBarOrientation()
+        InterfaceOrientation.Portrait
     )
 
+    /*
+     * On newest iOS interfaceOrientation will be deduced from [UIWindowScene] of [UIWindow]
+     * to which our [ComposeWindow] is attached.
+     * It's never UIInterfaceOrientationUnknown, if accessed after owning [UIWindow] was made key and visible:
```

Comment:
> It's never UIInterfaceOrientationUnknown

Why it's nullable then? Isn't it adding extra boxing?

Reply:
[Same rationale.](https://github.com/JetBrains/compose-multiplatform-core/pull/583#discussion_r1210494284)

You can write a code, that will end up with accessing this property and getting null. And we don't need to do anything about it.